easyaudit/tests/test_app/tests.py

```diff
@@ -24,14 +24,14 @@
 from easyaudit.middleware.easyaudit import set_current_user, clear_request
 
 
-TEST_USER_EMAIL = 'joe@example.com'
-TEST_USER_PASSWORD = 'password'
-TEST_ADMIN_EMAIL = 'admin@example.com'
-TEST_ADMIN_PASSWORD = 'password'
+class WithUserInfoMixin:
+    def setUp(self):
+        self.username = 'joe@example.com'
+        self.email = 'joe@example.com'
+        self.password = 'password'
 
 
-@override_settings(TEST=True)
-class TestDjangoCompat(TestCase):
+class TestDjangoCompat(SimpleTestCase):
```

Comment:
Didn't look like a database was needed for this test

Reply:
Indeed that is the case. I think it's simpler to use the standard class but agreed, we're actually asserting django logic, not reaching into any db connection requirements.